pkg/retry/retry.go

```diff
@@ -140,6 +141,33 @@ func (s *Server) handleRetry() http.Handler {
 				logger.Infow("redeliver failed event", "eventID", *event.ID, "guid", *event.GUID)
 
 				if err := s.github.RedeliverEvent(ctx, *event.ID); err != nil {
+					if _, ok := err.(*github.AcceptedError); ok {
+						logger.Infow("accepted by GitHub",
```

Comment:
Id make this message more descriptive:

`skipping redeliver event because it has already been submitted to GitHub`